e2e/rbd.go

```diff
@@ -484,6 +507,13 @@ var _ = Describe("RBD", func() {
 						go func(w *sync.WaitGroup, n int, p v1.PersistentVolumeClaim, a v1.Pod) {
 							name := fmt.Sprintf("%s%d", f.UniqueName, n)
 							wgErrs[n] = createPVCAndApp(name, f, &p, &a, deployTimeout)
+							filePath := appClone.Spec.Containers[0].VolumeMounts[0].MountPath + "/test"
+							checkSumClone := ""
+							checkSumClone, wgErrs[n] = calculateSHA512sum(f, &a, filePath, &opt)
+							e2elog.Logf("checksum value for the clone is %s", checkSumClone)
+							if checkSumClone != checkSum {
+								e2elog.Failf("checksum value did not match. Checksum=%v and checkSumClone=%v", checkSum, checkSumClone)
+							}
```

Comment:
if wgErrs[n]==nil && checkSumClone != checkSum{
wgErrs[n]=fmt.Errorf("checksum value did not match. Checksum=%v and checkSumClone=%v", checkSum, checkSumClone)
}